docs/development/pigs/pig-011.rst

```diff
@@ -0,0 +1,274 @@
+``Dataset`` itself and be serialized independently or be added as an extension to
+the serialized ``count`` data container (a ``CountsSpectrum`` or ``Map``), as is
+done for OGIP spectra. In order to stack several ``Dataset`` objects it is necessary
+to be able to combine ``GTI`` tables. While a simple stacking of tables is enough
+in many cases, the situation of overlapping time intervals should be considered and
+a proper ``GTI.union()`` should be introduced (a functionality recurrent in HEA
+software see e.g. ftools `mgtime)``
+
+
+Data Fitting
+------------
+
+The data fitting is the step were the ``Datasets`` are converted into a lightcurve
+based on a model of the emission. The amplitude of the source model is fitted while
+other parameters are usually, but not necessarily, fixed.
+
+The fitting is very similar to the extraction of ``FluxPoints`` in the spectral analysis
+and does not strongly depend on the type of ``Datasets`` considered. The new
+:class:``LightCurveEstimator`` class should therefore be able to take as input both
+``SpectrumDatasetOnOff`` and ``MapDataset``, in a similar fashion as the current
+``FluxPointEstimator``.
+
+After creating the ``Datasets``, the user would first define the model to be used
+and freeze its parameters. Then we would apply it to the various ``Dataset`` objects
+before calling the ``LightCurveEstimator``:
+
+.. code ::
+
+    sky_model = SkyModel(spatial_model=spatial_model, spectral_model=spectral_model)
+    sky_model.parameters["index"].frozen = True
+    sky_model.parameters["lon_0"].frozen = True
+    sky_model.parameters["lat_0"].frozen = True
+
+    for dataset in datasets:
+        dataset.model = sky_model
+
+    lc_estimator = LightCurveEstimator(datasets)
+    lightcurve = lc.run()
+
+To help, the user deal with model parameters some helper function could be introduced
+to freeze all parameters of a model. Another possibility is to assume that all parameters
+are fixed except the `amplitude` in the ``LightCurveEstimator.run()`` method, or pass as
+arguments the names of the parameters to leave free.
+
+Storing the results and further studies
+---------------------------------------
+
+The results are returned as a ``gammapy.time.LightCurve`` instance (the current
+container class for light curves) that, so far, essentially holds the integrated flux +
+errors and the time_min/time_max of each time bin. There are many other quantities
+which could be stored, such as the energy range of the flux, the number of excess
+events in the time bin considered, etc.
+The current container class already provides some methods to study variability, such as
+chi-square test, fractional variance estimation.
+
+Example usage:
+
+.. code ::
+
+    lightcurve.plot('amplitude')
+
+    # Get fractional variance
+    print(lightcurve.fvar('amplitude'))
+
+The ``LightCurve`` object should be able to rpovide some rebinning functionalities.
```

Comment:
I think this requires the likelihood profiles to be stored in the `LightCurve` object, which you currently don't mention yet.